packages/office-ui-fabric-react/src/components/SwatchColorPicker/ColorPickerGridCell.styles.ts

```diff
@@ -20,7 +20,12 @@ const cellHighContrastFocus = {
 
 export const getStyles = (props: IColorPickerGridCellStyleProps): IColorPickerGridCellStyles => {
   const { theme, disabled, selected, circle, isWhite, height = DEFAULT_CELL_SIZE, width = DEFAULT_CELL_SIZE, borderWidth } = props;
-  const { semanticColors } = theme;
+  const { semanticColors, palette } = theme;
+
+  const buttonBorderHovered = palette.neutralLighter;
```

Comment:
shouldn't these be semanticColors?

Reply:
Strange, I left a reply to this comment about a week ago but it's not here. Anyways @khmakoto told me he was addressing these colors in his work on the Button component and we agreed that he would add the colors in.

Reply:
@dzearing `buttonBorderHovered` doesn't exist at the moment in `semanticColors` (I don't know why). Since we are in the middle of talks to revamp our color system I thought it wasn't a good idea to add the color, but we can add it as part of this if you want.